src/components/App.test.js

```diff
@@ -8,15 +8,14 @@ test("renders Able Typeahead", () => {
   expect(linkElement).toBeInTheDocument();
 });
 
-test.skip("user can type on search bar", () => {
+test("user can type on search bar", async () => {
   render(<App />);
   const textInput = screen.getByPlaceholderText(/Search/i);
   expect(textInput).toBeInTheDocument();
   let resultsList = screen.queryByTestId("results-list");
   expect(resultsList).not.toBeInTheDocument();
   userEvent.type(textInput, "Spiderman");
-  resultsList = screen.queryByTestId("results-list");
-  expect(resultsList).toBeInTheDocument();
+  await waitFor(() => expect(screen.queryByTestId("results-list")).toBeInTheDocument());
```

Comment:
Since we "debounced" the search, I had to use `waitFor` to wait for the suggestions list to appear and disappear.